### DISCUSSION TITLE
I saw improvements on demo 1. Is it still needed?

### DISCUSSION THREAD

**Discussion by frederikhors** [jycouet/kitql#98] [Q&A] [answered]
I saw improvements on demo 1.

Is it still needed?

**Comment by jycouet** [comment, marked as answer]
I fixed a typo in the README.
I'm still on hold, waiting for you. If you can't / don't what to do it, just let me know and I will update de demo1 with the latest.

**Reply by frederikhors** [reply]
Ok. I think it's better if YOU add the two pages (list and details) from https://github.com/frederikhors/demo-kitql-sveltekit.

**Reply by jycouet** [reply]
ok, I will

**Reply by jycouet** [reply]
Should be good now 👌

2 Options
- Clone: https://kitql.vercel.app/docs/demos/01_demo-01
- stackblitz: <a target="_blank" href="https://stackblitz.com/fork/github/jycouet/kitql/tree/main/examples/demo1?title=KitQL%20Demo%201">
	<img src="https://developer.stackblitz.com/img/open_in_stackblitz.svg" alt="Open in StackBlitz" />
</a>

**Reply by frederikhors** [reply]
Amazing! I tried it on StackBlitz!
